src/pages/space/components/planspace/PlanSpace.tsx

```
@@ -4,46 +4,55 @@ import PlanCard from './PlanCard';
 import Add from '@/assets/icons/Add';
 import NewPlanWindow from './NewPlanWindow';
 import { useModal } from '@/hooks/useModal';
+import { useQuery } from '@tanstack/react-query';
+import axiosInstance from '@/api/axiosInstance';
```

ESLint

[error] 'axiosInstance' is defined but never used.

(@typescript-eslint/no-unused-vars)